gulpfile.js

```diff
@@ -0,0 +1,101 @@
+var _ = require('lodash');
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var ts = require('gulp-typescript');
+var tslint = require('gulp-tslint');
+var realTslint = require('tslint');
+var spawnSync = require('child_process').spawnSync;
+
+function build(src, dest) {
+  var tsProject = ts.createProject('configs/tsconfig-base.json', { declaration: true });
+  return gulp.src(src)
+    .pipe(tsProject())
+    .pipe(gulp.dest(dest));
+}
+
+gulp.task('build:capnp', function () {
```

Comment:
Let's call this task `build:capnp-ts` instead, since that matches the actual package name and it's not impossible we'll be introducing a `capnp-js` package down the road.